MovieFlix/app/src/main/res/layout/fragment_movie.xml

```diff
@@ -40,17 +39,33 @@
 
         </LinearLayout>
 
+        <EditText
+            android:id="@+id/search"
+            android:layout_width="match_parent"
+            android:layout_height="55dp"
+            android:layout_marginTop="5dp"
+            android:background="@color/grey"
+            android:textColorHint="@color/white"
+            android:textColor="@color/white"
+            android:hint="@string/search"
+            android:layout_marginBottom="15dp"
+            android:paddingStart="20dp"
+            android:textSize="22sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/linearLayout2"
+            app:layout_constraintBottom_toTopOf="@id/movie_recycler_view"/>
+
         <androidx.recyclerview.widget.RecyclerView
             android:id="@+id/movie_recycler_view"
-            android:layout_width="0dp"
-            android:layout_height="0dp"
+            android:layout_width="399dp"
+            android:layout_height="525dp"
             android:layout_marginStart="1dp"
```

Comment:
do not specify width and height... will cause issue in devices of different sizes